client/send-text.cc

```diff
@@ -162,10 +167,15 @@ int main(int argc, char *argv[]) {
     // dry-run to determine total number of pixels.
     const int total_len = DrawText(&display, font, 0, y_pos, fg, NULL, text);
 
+    // if rotated, center in in the available display space.
+    const int x_pos = (width ) / 2 ;
```

Comment:
Here is an extra space before the closing parenthesis. Also, we don't really need the parentheses at all.